### PR TITLE
Amc 202409

### PR DESCRIPTION
**Pull request by aoifem01**
Creating GP report outputs from R, plus small formatting changes for the 2024 Autumn run.

### CODE REVIEW

File: code/x_coverage_prisons.R
Comment (karen-hotopp) [inline code/x_coverage_prisons.R:40]: Come back to this. Check SPSS code and define this variable (in notes in script). Someone born 31Mar1958 is 66 today (28Oct2024).

File: code/x_master.R
Comment (karen-hotopp) [inline code/x_master.R:59]: Which part doesn't work? Can we have a quick discussion re: this script?

**Review by karen-hotopp — APPROVED** [review]

File: code/write_excel/95_Source_Excel_4.R
Comment (aoifem01) [reply code/write_excel/95_Source_Excel_4.R:82]: You're definitely right, I copied and pasted and then only changed the object name oops

File: code/write_excel/93_Source_Excel_2.R
Comment (aoifem01) [reply code/write_excel/93_Source_Excel_2.R:171]: These are 2 previous financial years, so 20WW/XX is the new data (2023/24), then 20UU/VV (2021/22) and 20VV/WW (2022/23) are the 2 years prior to that, which is what this note refers to?

File: code/write_excel/96_Write_Excel_4.R
Comment (aoifem01) [reply code/write_excel/96_Write_Excel_4.R:50]: Nope everything changed to fin_year

File: code/write_excel/93_Source_Excel_2.R
Comment (aoifem01) [reply code/write_excel/93_Source_Excel_2.R:171]: Change these back

File: code/7_4_vascular_referrals.R
Comment (aoifem01) [reply code/7_4_vascular_referrals.R:385]: It gets filtered in the second instance (376) to only include cases with result_outcome in either 15 or 16, whereas there's 1 result_outcome = 20 EVAR case in the extract

**Comment by aoifem01** [comment]
@karen-hotopp just to let you know the final commit is here to be approved - once done feel free to merge and delete!

File: code/7_4_vascular_referrals.R
Comment (karen-hotopp) [reply code/7_4_vascular_referrals.R:386]: Made into issue to further investigate.

File: code/91_2_coverage_gp_practice.R
Comment (karen-hotopp) [reply code/91_2_coverage_gp_practice.R:165]: Made into issue to investigate further.

**Pull request merged**
